### PR TITLE
A few changes allowing to send transforms over IGT link.

### PR DESCRIPTION
**Pull request by faustomilletari**
please refer to commit messages. 

I also had to change a few lines of the code that sends images over IGT link because the images were arriving incorrectly in 3D Slicer which uses IGT link to receive all kind of things. 

I also added some more parameters to the interfaces to be able to names for the stuff that we send and to specify the network interface we want to bind the server to.

### CODE REVIEW

**Comment by faustomilletari** [comment]
I think that you will have to solve the test failures by yourself. sorry about that.

**Comment by faustomilletari** [comment]
Hey,

I have noticed the test failure. I have no idea how to solve it but without that line in init it does not work when I try to do import pyIGTLink after installing via pip. Suggestions?!

File: pyIGTLink/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from pyIGTLink import *
```
Comment (Danielhiversen) [inline pyIGTLink/__init__.py]: Add `# noqa: F403 F401` in the end of the line to fix the test

**Comment by Danielhiversen** [comment]
Great!!
Will merge when tests are passing.

**Pull request merged**